_data/taps/extraction/database-setup/server-settings/mysql.yml

```diff
@@ -61,6 +61,16 @@ defaults:
       **Note**: This is applicable when using a read replica.
 
 
+# -------------------------- #
+#       AZURE SETTINGS       #
+# -------------------------- #
+
+microsoft-azure-mysql:
```

Comment:
I think you might have forgotten to come back to this. There are other parameters that should be in here. Take a look at the lists for the other integrations in this file - what do they have that this list doesn't?